### DISCUSSION TITLE
Not able to run in DigitalOcean App Platform

### DISCUSSION THREAD

**Discussion by dlidstrom** [umbraco/Umbraco-CMS#13604] [Features and ideas]
I've been trying to run Umbraco in DigitalOcean App Platform. This platform is K8s, fully managed and abstracted to make things easy. However it seems the installation flow is tripping up App Platform. I am unable to load the installation wizard as App Platform responds with

```
We encountered an error when trying to load your application and your page could not be served.
Check the logs for your application in the App Platform dashboard.
```

I *think* the reason is this:

https://github.com/umbraco/Umbraco-CMS/blob/v11/contrib/src/Umbraco.Web.BackOffice/Install/InstallController.cs#L60

The last lines logged in runtime logs in App Platform is just that:

```txt
[21:58:06 INF] Request finished HTTP/1.1 GET http://xxxxx.ondigitalocean.app/ null null - 503 - ["Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.1 Safari/605.1.15"] 302.8777ms
```

How is the install flow supposed to work? Returning 503 and expecting the browser to render `UmbracoInstall/Index.cshtml`?
@Zeegaan

**Comment by bergmania** [comment]
Hi @dlidstrom.

The installer is skipped if you use [unattended installs](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/install/unattended-install) and upgrades. 

Please try if this works

**Comment by dlidstrom** [comment]
Hi @bergmania! That worked like a charm, thanks!
